doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -492,6 +492,10 @@ Modem libraries
 
   * Added a log warning suggesting a SIM card to be installed if a UICC error is detected by the modem.
 
+* :ref:`nrf_modem_lib_socket` library:
```

Comment:
This will convert to `'socket offloading' library` Is that what we want or shold it be `nrf_modem_lib_readme` instead so that it becomes `Modem library integration layer`? 

I've not seen `:ref:'nrf_modem_lib_socket'` used elsewhere in the changelog.

Reply:
Updated to `nrf_modem_lib_readme`.